src/rpc/handlers/Tx.cpp

```diff
@@ -36,6 +36,18 @@ doTx(Context const& context)
     if (!range)
         return Status{RippledError::rpcNOT_READY};
 
+    if (request.contains(JS(min_ledger)) && request.contains(JS(max_ledger)) &&
+        request.at(JS(min_ledger)).is_int64() &&
```

Comment:
I'd add these `is_int64()` checks inside instead and return some error code if they are passed but not of the correct type.

Reply:
@godexsoft added a new ClioError code, rpcMALFORMED_PARAMS, as a counterpart to the rippled INVALID_PARAMS. Functionally in rippled, invalidParams was passed when unknown parameters were input and there were very specific parameters with malformed error codes

Seeing as min and max ledger are used locally, I considered this merited a more universal error code, with the hardcoded error string a more specific explanation as to why. Major parameters like accounts, currency types, etc. should use the more specific error codes rippled passes by default 
![image](https://user-images.githubusercontent.com/47958858/202068790-2d108d75-5263-4617-b7fe-e414b64e7e9b.png)

Reply:
Update to the above @godexsoft: found a specific error code for malformed ledger indexes in rippled, but kept rpcMALFORMED_PARAMS in Errors.h for future use. Lmk if you want me to delete it otherwise